dist/cloudfront/get.openziti.io/routes.yml

```diff
@@ -38,3 +38,7 @@
 - get: /zdew/
   raw: /openziti/desktop-edge-win/main/release-streams/
   file: latest.json
+
+- get: /minimal/
```

Comment:
are we capable of using something like `/dock/minimal` here? (two path segments)? I'd like to see this under the existing `dock`

Reply:
I removed the additional shortcut route. The existing directory shortcut `get.openziti.io/dock` to `/quickstart/docker` in this repo already provides a getter URL like get.openziti.io/dock/minimal/compose.yml`.